### PR TITLE
fix(types): add random function to sequelize

### PR DESCRIPTION
**Pull request by Chocobozzz**
### Pull Request check-list

_Please make sure to review and check all of these items:_

- [x] Does `npm run test` or `npm run test-DIALECT` pass with this change (including linting)?
- [x] Does the description below contain a link to an existing issue (Closes #[issue]) or a description of the issue you are solving? **no**
- [x] Have you added new tests to prevent regressions?
- [x] Is a documentation update included (if this change modifies existing APIs, or introduces new ones)? **no**
- [x] Did you follow the commit message conventions explained in [CONTRIBUTING.md](https://github.com/sequelize/sequelize/blob/master/CONTRIBUTING.md)?

### CODE REVIEW

File: types/test/sequelize.ts [hunk 1]
```diff
@@ -45,4 +45,6 @@ Sequelize.afterConnect(() => {
 
 });
 
+sequelize.random()
```
Comment (SimonSchick) [inline types/test/sequelize.ts]: Should be `const rnd: Fn = ...;`

File: types/lib/sequelize.d.ts [hunk 2]
```diff
@@ -1123,6 +1123,11 @@ export class Sequelize extends Hooks {
   public query<T extends object>(sql: string | { query: string; values: unknown[] }, options: QueryOptionsWithType<QueryTypes.SELECT>): Promise<T[]>;
   public query(sql: string | { query: string; values: unknown[] }, options?: QueryOptions | QueryOptionsWithType<QueryTypes.RAW>): Promise<unknown[]>;
 
+  /**
+   * Get the fn for random based on the dialect
+   */
+  public random(): Fn
```
Comment (SimonSchick) [inline types/lib/sequelize.d.ts]: Missing semicolon

**Comment by Chocobozzz** [comment]
Thanks @SimonSchick and sorry for the late.

**Pull request merged**